static_frame/core/util.py

```diff
@@ -3186,5 +3187,27 @@ def iloc_to_insertion_iloc(key: int, size: int) -> int:
         raise IndexError(f'index {key} out of range for length {size} container.')
     return key % size
 
-
-
+def total_getsizeof(it: tp.Iterable[any], *, seen=None) -> int:
```

Comment:
We try to use "hierarchical naming", meaning you name from the general to more specific, which may not always be obvious, but can be really helpful for consistency and intuitions about what things are named. I would consider naming this function `'getsizeof_total()` or `getsizeof_recursive()`, putting the `getsizeof` bit first.

Reply:
Renamed it to `getsizeof_recursive` to follow this convention.